### PR TITLE
Hot fixes

### PR DESCRIPTION
**Pull request by Zeeerooo**

### CODE REVIEW

**Comment by Phfollert** [comment]
LGTM

File: app/report/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
-from . import views
\ No newline at end of file
+from app.report import views
+from app.report import antenna_signal_report_generation
+from app.report import general_report_generation
+from app.report import reports_generation
```
Comment (sebablasko) [inline app/report/__init__.py]: no newline at end... it follows pep8?

Comment (Zeeerooo) [inline app/report/__init__.py]: Sorry 😞

**Comment by sebablasko** [comment]
LGTM

**Pull request merged**